<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oknoRozmowy</class>
 <widget class="QDialog" name="oknoRozmowy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushWyslij">
      <property name="text">
       <string>Wyslij</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="licznikiZnakow">
      <property name="text">
       <string>0 / 256</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushZakoncz">
      <property name="text">
       <string>Zako≈Ñcz</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>511</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="oknoWiadomosci"/>
    </item>
    <item>
     <widget class="QLineEdit" name="liniaWiadomosci"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
